test/validation/api/packet/packet.c

```diff
@@ -2354,7 +2354,7 @@ void packet_test_ref(void)
 	ref_pkt[0] = odp_packet_ref(base_pkt, offset[0]);
 	CU_ASSERT_FATAL(ref_pkt[0] != ODP_PACKET_INVALID);
 	ref_len[0] = odp_packet_len(ref_pkt[0]);
-	CU_ASSERT(ref_len[0] = odp_packet_len(base_pkt) - offset[0]);
+	CU_ASSERT(ref_len[0] == odp_packet_len(base_pkt) - offset[0]);
```

Comment:
I would add just additional braces for (odp_packet_len(base_pkt) - offset[0])

Reply:
@muvarov that would be still an error, wouldn't it?

Reply:
I think now but in other places it's the same style. So changes are not needed.